cmudb/tscout/tscout.py

```diff
@@ -242,15 +249,18 @@ def processor(ou, buffered_strings):
         print("Processor for {} shut down.".format(ou.name()))
 
 
-if __name__ == '__main__':
-    # Parse the command line args, in this case,
-    # just the postmaster PID that we're attaching to.
-    if len(sys.argv) < 2:
-        print("USAGE: tscout PID")
-        exit()
-    pid = sys.argv[1]
 
-    setproctitle.setproctitle("{} TScout".format(pid))
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="TScout")
+    parser.add_argument("pid", type=int, help="Postmaster PID that we're attaching to")
+    parser.add_argument("--outdir", required=False, help="Output directory")
+    args = parser.parse_args()
+    pid = args.pid
+    outdir = args.outdir
+
+    setproctitle.setproctitle("{} TScout Coordinator".format(pid))
 
```

Comment:
nitpick proposal: handle creation of outdir somewhere around here,

```
if outdir is None:
    outdir = '.'
else:
    Path(outdir).mkdir(parents=True, exist_ok=True)
```

more generally, if it is possible to push setup logic out of the TScout components (so that the code in those is as simple as possible), it might be nice to do that